optaplanner-operator/optaplanner-operator-integration-test-parent/school-timetabling-parent/kubernetes-demo-common/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>school-timetabling-parent</artifactId>
+    <groupId>org.optaplanner</groupId>
+    <version>8.34.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>kubernetes-demo-common</artifactId>
```

Comment:
There is an inconsistency among the modules (and artifacts) names:  `school-timetabling-parent` vs. `kubernetes-demo-common` and `kubernetes-demo-school-timetabling`

I would pick just one name that fits its purpose the best.